userspace/ksud/Cargo.toml

```diff
@@ -23,8 +23,12 @@ encoding = "0.2.33"
 retry = "2.0.0"
 humansize = "2.0.0"
 libc = "0.2"
+sys-mount = "2.0.1"
+android-properties = "0.2.2"
```

Comment:
use `android-properties = { version = "0.2.2", features = ["bionic-deprecated"] }`, otherwise Android 8- could crash because there isn't `__system_property_read_callback`

```
  = note: ld: error: undefined symbol: __system_property_read_callback
          >>> referenced by ksud.99893b3e-cgu.4
          >>>               /Users/weishu/dev/github/KernelSU/userspace/ksud/target/aarch64-linux-android/release/deps/ksud-524741fa94bc2883.ksud.99893b3e-cgu.4.rcgu.o:(ksud::utils::getprop::hf4df51ac6a9f7de9)
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```